packages/db/src/database.ts

```diff
@@ -1,9 +1,37 @@
 import Knex from 'knex';
 
+import { RecordDoesNotExist } from '@cumulus/errors';
+
+import { tableNames } from './tables';
+
 export const isRecordDefined = <T>(record: T) => record !== undefined;
 
 export const doesRecordExist = async<T>(
   params: Partial<T>,
   knex: Knex,
   tableName: string
 ): Promise<boolean> => isRecordDefined(await knex<T>(tableName).where(params).first());
+
+/**
+ * Retrieve cumulus_id for a record from the specified table.
+ *
+ * @param {Object} whereClause - where clause for query
+ * @param {string} table - Name of table
```

Comment:
This is a nit but should the comment be `@param {tableNames} table - Name of table`? 
```suggestion
 * @param {tableNames} table - Name of table
```

Reply:
Agreed and changed